lib/vendor/stripe.js

```diff
@@ -0,0 +1,8 @@
+const stripe = require('stripe');
+
+if(process.env.NODE_ENV !== 'production'){
+  // NB: Stripe key for development and testing.
+  return module.exports = stripe('sk_test_W6L09JRZ1YR4Ua0KuDCDTST3');
```

Comment:
Should be removed and turned into a configuration option or an environment variable.